typed-racket-lib/typed-racket/types/overlap.rkt

```diff
@@ -113,7 +113,7 @@
     #f]
    ;; these are single-valued sequences
    [((Sequence: (list _ _ _ ...))
-     (or (? Pair?) (? Vector?) (? VectorTop?)))
+     (or (? Pair?) (Vector: _) (VectorTop:)))
     #:no-order
```

Comment:
Is there a reason to favor match patters w/ predicates vs trivial match expanders? just curious ;-) (I'm probably not consistent myself with this choice, lol)